test/Worker.Extensions.Shared.Tests/Reflection/ParameterBinderTests.cs

```diff
@@ -86,6 +98,19 @@ private static async IAsyncEnumerable<object> GetIntEnumerable(Type t)
             }
         }
 
+        private static async IAsyncEnumerable<object> GetPocoArray(Type t)
```

Comment:
this method was not really needed - could use `GetIntEnumerable` still. But no harm leaving it in if you want.